src/components_redux/Radios.js

```diff
@@ -9,10 +9,9 @@ export default class Radios extends Component {
   radioChange = event => {
     this.setState(
       {
-        type: event.target.value,
-        selectedValue: event.target.value
+        type: event.target.value
       },
-      () => this.props.handleChangeShow(this.state.selectedValue)
+      this.props.handleChangeShow(event.target.value)
```

Comment:
If I see it right, its still inside the setState method?
This should even lead to an error.

Reply:
Yes, it is, but there is no error.